src/drawing/DrawingAnnotations.tsx

```diff
@@ -29,6 +31,8 @@ export type Props = {
     setupDrawing: (location: number) => void;
     stashedPathGroups: Array<PathGroup>;
     undoDrawingPathGroup: () => void;
+    targetType: typeof TARGET_TYPE.PAGE | typeof TARGET_TYPE.FRAME;
```

Comment:
Can you not just make this type `TARGET_TYPE`?

Reply:
Yup. Didn't catch that when I converted everything to use the enums